zammad/templates/cronjob-reindex.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "zammad.fullname" . }}-cronjob-reindex
+  labels:
+    {{- include "zammad.labels" . | nindent 4 }}
+    app.kubernetes.io/component: zammad-cronjob-reindex
+  annotations:
+    description: |-
+      This cronjob never runs. It can be used to create maintenance jobs with
```

Comment:
Could we change this to "never runs or default"

Reply:
Removed